packages/plugin-bootstrap/src/providers/providers.ts

```diff
@@ -20,42 +20,53 @@ export const providersProvider: Provider = {
   name: 'PROVIDERS',
   description: 'List of all data providers the agent can use to get additional information',
   get: async (runtime: IAgentRuntime, _message: Memory, _state: State) => {
+    const allProviders = runtime.providers;
+
     // Filter providers with dynamic: true
-    const dynamicProviders = runtime.providers.filter((provider) => provider.dynamic === true);
+    const dynamicProviders = allProviders.filter((provider) => provider.dynamic === true);
 
     // Create formatted text for each provider
-    const providerDescriptions = dynamicProviders.map((provider) => {
+    const dynamicDescriptions = dynamicProviders.map((provider) => {
       return `- **${provider.name}**: ${provider.description || 'No description available'}`;
     });
 
-    // Create the header text
-    const headerText =
-      '# Providers\n\nThese providers are available for the agent to select and use:';
+    const allDescriptions = allProviders.map((provider) => {
+      return `- **${provider.name}**: ${provider.description || 'No description available'}`;
```

Comment:
Maybe we shouldn’t include them and complain loudly in the logs if no description

Reply:
yea should remove it 🥲